samples/client/petstore/java/default/src/main/java/io/swagger/client/model/Pet.java

```diff
@@ -6,6 +6,7 @@
 import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
 import io.swagger.client.model.Category;
+import java.util.*;
```

Comment:
No worry. Next Java PR should take care of it (as generating the petstore sample should be done before submitting the PR)